packages/git/src/browser/history/git-history-contribution.ts

```diff
@@ -35,9 +35,8 @@ export namespace GitHistoryCommands {
     export const OPEN_FILE_HISTORY: Command = {
         id: 'git-history:open-file-history',
     };
-    export const OPEN_BRANCH_HISTORY: Command = {
-        id: 'git-history:open-branch-history',
-        label: GIT_HISTORY_LABEL
+    export const TOGGLE_BRANCH_HISTORY: Command = {
```

Comment:
Can we keep the old name?
There may be instances of external extensions calling `executeCommand` with the old const.
This means that those callers will be broken.

Reply:
Ok, I will modify this tomorrow.

Reply:
Why do you suggest we have two commands?

Reply:
Is there a command to open branch history when the content of git history is file history?

Reply:
From what I understand, the command `Git History` displays the history for a given branch.
If you want the `Git History` for a file, the context menu is used (example: from the explorer, right click a file, and select `Git History`). 

Is there more that should be added?

Reply:
Ok, thank you for your explanation, I understand. I will update this commit to keep the old const name and enable the UI test for git history tomorrow.

Reply:
done.